src/py42/clients/securitydata.py

```diff
@@ -172,19 +172,38 @@ def get_all_user_security_events(
         )
 
     def search_file_events(self, query):
-        """Searches for file events.
+        """Searches for file events, returns up to the first 10,000 events.
         `REST Documentation <https://support.code42.com/Administrator/Cloud/Monitoring_and_managing/Forensic_File_Search_API>`__
 
         Args:
-            query (:class:`py42.sdk.queries.fileevents.file_event_query.FileEventQuery`): Also
-                accepts a raw JSON str.
+            query query (`:class:`py42.sdk.queries.fileevents.file_event_query.FileEventQuery` or str): The file event query to filter search results.
 
         Returns:
             :class:`py42.response.Py42Response`: A response containing the first 10,000
             events.
         """
         return self._file_event_service.search(query)
 
+    def search_all_file_events(self, query, page_token=""):
+        """Searches for all file events, returning a page of response with a token in the response to retrieve next page.
+
+        `REST Documentation <https://support.code42.com/Administrator/Cloud/Monitoring_and_managing/Forensic_File_Search_API>`__
+
+        Args:
+            query query (`:class:`py42.sdk.queries.fileevents.file_event_query.FileEventQuery` or str): The file event query
+                to filter search results.
+            page_token (str): A token used to indicate the starting point for additional page results. Defaults to empty string.
+                For the first page do not pass page_token, for all consecutive pages pass the token returned from previous response,
```

Comment:
The punctuation is a little off here.

`For the first page, do not pass page_token.  For all consecutive pages, pass the token returns from the previous response from the field `nextPgToken`.

Reply:
Changed, rephrased `pass the token returns from the previous...` to `pass the token from the previous...`